resources/configuration_file_template.yml

```diff
@@ -1,10 +1,13 @@
+# You can remove these two lines to have credentials read from
+# the environment variables AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY or
+# ~/.aws/credentials.
 s3_id: YOUR_AWS_S3_ACCESS_KEY_ID
 s3_secret: YOUR_AWS_S3_SECRET_ACCESS_KEY
-s3_bucket: your.blog.bucket.com
-
 # set s3_token if using temporary credentials with a session token (eg: when assuming a role)
 # s3_token: YOUR_AWS_S3_SESSION_TOKEN
```

Comment:
You're right! Looks like this changed in https://github.com/laurilehmijoki/s3_website/commit/0ef15b202a45b7d12df7f2622ebc85aec525d1f0. @laurilehmijoki would you like a fix in this PR, or do you want to fix this on master?